src/libs/ReportActionsUtils.ts

```diff
@@ -73,6 +73,19 @@ Onyx.connect({
     callback: (val) => (isNetworkOffline = val?.isOffline ?? false),
 });
 
+let currentUserAccountID: number | undefined;
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    callback: (value) => {
+        // When signed out, val is undefined
+        if (!value) {
```

Comment:
NAB - can we match up `val` and `value`? Use one or the other in the code & comment? 🙏

Reply:
fixed the comment